### ISSUE TITLE
I want to update the same keras model with two different losses by train_on_batch methods

### ISSUE THREAD

**Issue by TanmDL** [keras-team/keras#14588]
Example.

**MLP is a simple functional API.** 

mlp1=MLP() 
mlp1.compile(loss=loss1, optimizer=adamOpt)

mlp2=MLP() 
mlp2.compile(loss=loss2, optimizer=adamOpt)

for i in batch: 
     mlp1.train_on_batch(data[i], label[i]) 
     mlp2.train_on_batch(data[i], label[i])

The two losses will be updated weights of MLP model. This idea can be easily implemented into PyTorch and it is working fine. 
Am I doing correctly or not?

**Comment by jvishnuvardhan** [comment]
@TanmDL Sorry for the late response. Did you try training two models (of same architecture) but with different losses? The above approach is feasible even with `keras`. Please check the [guide](https://keras.io/guides/functional_api/) on keras website.

If you face any difficulties, please post a standalone code to reproduce the issue. You could also post them on [TF Forum](https://discuss.tensorflow.org/tag/help_request) where response would be faster for support kind of questions. GitHub is mainly for bugs and performance related issues. Thanks!
